src/main/kotlin/process/pitch/UtauMode1PitchConversion.kt

```diff
@@ -0,0 +1,68 @@
+package process.pitch
+
+import io.UstMode1
+import model.Note
+import model.Pitch
+import process.interpolateLinear
+
+data class UtauMode1TrackPitchData(
+    val notes: List<UtauMode1NotePitchData?>
+)
+
+/** This class contains Utau Pitch Data in Mode1.
+ * However, it ignores some parameters which are contained in the ust(mode1) file in purpose:
+ *
+ * PBType: it indicates which data format this file uses. It will always be treated as "5"
+ * since UTAU only uses "OldData" in very early versions.
+ *
+ * PBStart: it indicates when the pitch data begins, and it will be negative when pitch data
+ * begins from PreUtterance of the note, otherwise, it will be 0. Since Only old versions of utau
+ * produce this kind of data, and it will not save this value in the ust (this field is introduced
+ * in version 0.4.0), presuming it as 0 will make things much easier.
+ *
+ * Please notice that UTAU save its pitch data by cent, not semitone. This class will keep this behaviour.
+ */
+data class UtauMode1NotePitchData(
+    val pitchData: Pitch?
```

Comment:
`Pitch` is only used in `Project`, and it's project level, so here you can just use the data points (List<Pair<...>>)